addons/mail/static/src/components/chat_window_header/chat_window_header.js

```diff
@@ -46,10 +50,24 @@ class ChatWindowHeader extends Component {
      * @param {MouseEvent} ev
      */
     _onClick(ev) {
+        if (isEventHandled(ev, 'ChatWindowHeader.openProfile')) {
+            return;
+        }
         const chatWindow = this.chatWindow;
         this.trigger('o-clicked', { chatWindow });
     }
 
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onClickChatHeaderName(ev) {
```

Comment:
`_onClickChatHeaderName` => `_onClickName`
The `ChatHeader` part is redundant with name of component.